portal/templates/flask_user/emails/eproms_base_message.html

```diff
@@ -6,3 +6,6 @@
 <p>Thank you,</p>
 <p>The TrueNTH Team</p>
 {% endtrans %}
+{% trans parent_org=user.first_top_organization() %}
```

Comment:
My mistake, this will return the top-level org (ie Ironman/CRV).

You'll just want to use the first org name in the way below:
```suggestion
{% trans parent_org=user.organizations[0].name %}
```
---
`flask shell` example:
```
>>> from portal.models.user import User
>>> User.query.get(1).organizations[0].name
u'Centro de Pesquisa em Oncologia'
```

Reply:
@ivan-c  thanks for testing! I actually do want the top level org; I have just changed that to reference it's name, tho.